.ci/gitlab/.gitlab-ci.yml

```
@@ -160,7 +183,7 @@ default:
     - (Get-WmiObject Win32_PhysicalMemory | measure-object Capacity -sum).sum/1kb
     - $ErrorActionPreference=$ErrorActionOld
     - *clone_spack
-    - . .\spack\share\spack\setup-env.ps1
+    - . $SPACK_ROOT\share\spack\setup-env.ps1
```

This is set in the environment, not in the shell right?
If so needs to be:
```suggestion
    - . $Env:SPACK_ROOT\share\spack\setup-env.ps1
```